corehq/apps/reports/templates/reports/filters/xpath_textarea.html

```diff
@@ -0,0 +1,41 @@
+{% extends 'reports/filters/base.html' %}
+{% load hq_shared_tags %}
+{% block filter_content %}
+    <div class="report-filter-xpath-textarea" data-allcaseproperties="{{ all_case_properties }}">
+    {% if help_title and help_content %}
+        <div class="row">
+            <div class="col-xs-11">
+    {% endif %}
+
+    <textarea
+        id="{{ css_id }}"
+        name="{{ slug }}"
+        type="text"
+        spellcheck="false"
+        style="resize: vertical; min-height: 60px;"
+        {% if required %}required="true"{% endif %}
+        {% if placeholder %}placeholder="{{ placeholder }}"{% endif %}
```

Comment:
Might want to use `html_attr` just so no one looks at this and wonders about escaping issues.